app/views/results/common/_flexible_marks_summary_table.html.erb

```diff
@@ -26,7 +26,7 @@
     <tr class="<%= cycle('mark_summary_holderA', 'mark_summary_holderB') %>" id="<%= "test_script_summary" %>">
         <td><%=t("summary_table.test_script_title")%></td>
         <td><%= h(@assignment.total_test_script_marks * 1.0) %></td>
-        <td><span id="test_script_summary_mark"><%= h(@submission.result.get_total_test_script_marks * 1.0) %></span> / <%=  h(@assignment.total_test_script_marks * 1.0) %></td>
+        <td><span id="test_script_summary_mark"><%= h(@result.get_total_test_script_marks * 1.0) %></span> / <%=  h(@assignment.total_test_script_marks * 1.0) %></td>
```

Comment:
Was there a reason for this change besides just cleaning up the code?

I approve of the change, but would have preferred to see it in a separate pull request to master if it is independent of the test-framework project.